src/Symfony/Component/VarDumper/Tests/Fixtures/ExtendsReflectionTypeFixture.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Component\VarDumper\Tests\Fixtures;
+
+class ExtendsReflectionTypeFixture extends \ReflectionType
+{
+    public function allowsNull() : bool
+    {
+        return false;
+    }
+
+    public function isBuiltin() : bool
+    {
+        return false;
+    }
+
+    public function __toString() : string
+    {
+        return 'fake';
+    }
```

Comment:
```suggestion
    public function allowsNull(): bool
    {
        return false;
    }

    public function isBuiltin(): bool
    {
        return false;
    }

    public function __toString(): string
    {
        return 'fake';
    }
```

Reply:
This is in the current commit.  I had left them untyped to preserve the parent interface I realize now that was unnecessary.

Reply:
Thanks!